samples/client/petstore/elixir/test/outer_enum_test.exs

```diff
@@ -6,6 +6,7 @@ defmodule OuterEnumTest do
 
   @valid_json """
   {
+    "enum_string_required": "lower",
```

Comment:
This shows that the previous implementation was "off", and shows one of the benefits of migrating to Ecto. Now, it's easier and more idiomatic to write proper type-casting and validations, which allows us to have a better Elixir generator.